test/yokozuna_schema_tests.erl

```diff
@@ -192,3 +231,48 @@ predefined_schema() ->
                                             {datatype, directory}
                                        ]}),
     {[], [Mapping], []}.
+
+%% Ensure that the riak_core_schema_tests module is loaded and return the
+%% path of the riak_core.schema file.
+riak_core_schema() ->
+    riak_core_schema(riak_core_dir()).
+riak_core_schema({RCDir, Schema}) when erlang:is_list(RCDir) ->
+    case code:ensure_loaded(riak_core_schema_tests) of
+        {module, _} ->
+            {true, Schema};
+        _ ->
+            Search = filename:join([RCDir, "**", "riak_core_schema_tests.beam"]),
+            case filelib:wildcard(Search) of
+                [Beam | _] ->
+                    case code:load_abs(filename:rootname(Beam)) of
+                        {module, _} ->
+                            {true, Schema};
+                        Error ->
+                            Error
+                    end;
+                [] ->
+                    {error, enoent}
+            end
+    end;
+riak_core_schema(Error) ->
+    Error.
+
+riak_core_dir() ->
```

Comment:
Instead of copy-pasting the same riak_core test code across multiple repos, could we put this in some kind of common test util module in riak_core?